contracts/TalentLayerService.sol

```diff
@@ -131,6 +164,28 @@ contract TalentLayerService is Initializable, ERC2771RecipientUpgradeable, UUPSU
         uint256 expirationDate
     );
 
+    /**
+     * @notice Emitted after a new proposal is created
+     * @param serviceId The service id
+     * @param ownerId The talentLayerId of the seller who made the proposal
+     * @param dataUri token Id to IPFS URI mapping
+     * @param status proposal status
+     * @param rateAmount the amount of token chosen
+     * @param platformId the platform ID on which the proposal was created
+     * @param expirationDate the timeout for the proposal
+     * @param referrerId the id of the referrer (Zero if no referrer)
+     */
+    event ProposalCreatedWithoutToken(
```

Comment:
Remove the two new event ProposalCreatedWithoutToken and ProposalUpdatedWithoutToken
and use a ProposalReferrerIdUpdated with only two params